extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -256,6 +300,16 @@ export class AmpGeo extends AMP.BaseElement {
         });
   }
 
+  /**
+   * Ensure that subdivisions allowed for a specific country
+   * @param {string} country
+   * @return {boolean}
+   * @private
+   */
+  areSubdivisionsAllowed_(country) {
+    return countries.CA === country || countries.US === country;
```

Comment:
Since this enum has only one intent, let's  rename `countries` to `countriesWithSubdivision`, and use `isEnumValue`

```js
import {isEnumValue} from '#core/types';
```
```suggestion
    return isEnumValue(countriesWithSubdivision, country);
```

Reply:
I made a commit with this changes, thank you!
@alanorozco @erwinmombay 
And I didn’t check it in a real app, mostly worked with tests, is it ok?
And how about amp-iso-subdivision- class support for prerender mode, I think it should be added on the server-side?
Do you do some tests after merge?




[10:16](https://amphtml.slack.com/archives/D034RL8GVSM/p1645647363338149)
And how about amp-iso-subdivision- class support for prerender mode, i think it should be added on server-side?

Reply:
Can you give me some advice How could I do some tests(real using)?

Reply:
I don't think we need a real end-to-end test, just a unit test for the pre-rendered subdivision that's not `us-ca`.

Reply:
I added a test for ca ca-mb for pre-rendered mode, and also checked that matched groups don't add as classes

Reply:
@jridgewell i'd need to implement the prerendering of `amp-iso-subdivision-${subdivision}` correct?